src/javascript/app_2/App/Components/Elements/Calendar/calendar_header.jsx

```diff
@@ -21,10 +21,10 @@ import {
 const CalendarHeader = ({
     calendar_date,
     calendar_view,
+    disabled_selector = [],
```

Comment:
You don't need to declare the `prop` as an empty array, add `PropType.Array` instead for `CalendarHeader`.

Reply:
thanks for pointing that out. i added proptypes.array for disabled_selector, but i think we still need to initialize empty array as a default value for disabled_selector or else, when its not passed, it read as null, instead of empty array

Reply:
@cakasuma please check conflicts, this prop is already fixed in master